backend/app/usecase/search/filter_test.go

```diff
@@ -0,0 +1,65 @@
+package search
+
+import (
+	"testing"
+
+	"github.com/short-d/app/fw/assert"
+	"github.com/short-d/short/backend/app/usecase/search/order"
+)
+
+func TestCreatedTime_ArrangeShortLinks(t *testing.T) {
```

Comment:
```suggestion
func TestNewFilter(t *testing.T) {
```

Reply:
used it as template 😓 . Thanks for the catch.